templates/MetadataGenerator/MD_DataIdentification.xml.twig

```diff
@@ -65,6 +65,21 @@
             }
         %}
     </gmd:descriptiveKeywords>
+    <gmd:descriptiveKeywords>
+        {% set anzsrc_keywords = [] %}
+        {% for keyword in dataset.datasetSubmission.keywords %}
+            {% if keyword.type.value == 'anzsrc' %}
```

Comment:
Is there a better way to handle enum constants?

Reply:
Yes, the twig constant function: `constant('App\\Enum\\KeywordType::TYPE_ANZSRC') `

Reply:
Also, value is not needed.